static/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -57,76 +55,68 @@ function EventOrGroupExtraDetails({
   const showReplayCount =
     organization.features.includes('session-replay-ui') && projectSupportsReplay(project);
 
-  const counts = useReplaysCount({
-    groupIds: id,
-    organization,
-    project,
-  });
```

Comment:
This was doing basically an N+1 query, because this component lives inside each row of all the Issues tables. 

This PR updates all the callsites so we do one query for all the table-rows that are visible.